controllers/api-endpoint.controller.js

```diff
@@ -0,0 +1,12 @@
+const {selectApiEndpoints} = require('../models/api-endpoint-model')
+
+exports.getApiEndpoints = (req, res, next) => {
+    selectApiEndpoints()
+    .then((response) => {
+        console.log(response)
+        res.status(200).send(response)
```

Comment:
Endpoints could be sent back in an object on a key of 'endpoints', but it's also fine as it is. I leave it up to you to refactor it or not.